src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimantResponseService.java

```diff
@@ -164,23 +165,52 @@ private void formaliseResponseAcceptance(
         }
     }
 
-    private void raiseAppInsightEvents(Claim claim, Response response, ClaimantResponse claimantResponse) {
+    private void raiseAppInsightEvents(Claim claim,
+                                        Response response,
+                                        ClaimantResponse claimantResponse,
+                                        CaseEvent caseEvent) {
+
         if (claimantResponse instanceof ResponseAcceptation) {
             appInsights.trackEvent(CLAIMANT_RESPONSE_ACCEPTED, REFERENCE_NUMBER, claim.getReferenceNumber());
+
         } else if (claimantResponse instanceof ResponseRejection) {
             if (isPartAdmissionOrIsStatePaidOrIsFullDefence(response)) {
-                raiseAppInsightEventForLegalAdvisorPilot(claim);
+                raiseAppInsightEventForOnlineOrOfflineDQ(claim);
                 raiseAppInsightEventForMediation(claim, response, (ResponseRejection) claimantResponse);
+
+                if (caseEvent != null) {
+                    raiseAppInsightsEventForPilot(claim, caseEvent);
+                }
             }
         } else {
             throw new IllegalStateException("Unknown response type");
         }
     }
 
-    private void raiseAppInsightEventForLegalAdvisorPilot(Claim claim) {
-        AppInsightsEvent appInsightsEvent = FeaturesUtils.isLegalAdvisorPilot(claim)
-            ? AppInsightsEvent.LA_PILOT_ELIGIBLE
-            : AppInsightsEvent.NON_LA_CASES;
+    private void raiseAppInsightsEventForPilot(Claim claim, CaseEvent caseEvent) {
+        switch (caseEvent) {
+            case ASSIGNING_FOR_LEGAL_ADVISOR_DIRECTIONS:
+                appInsights.trackEvent(AppInsightsEvent.LA_PILOT_ELIGIBLE, REFERENCE_NUMBER,
+                    claim.getReferenceNumber());
+                break;
+            case ASSIGNING_FOR_JUDGE_DIRECTIONS:
+                appInsights.trackEvent(AppInsightsEvent.JDDO_PILOT_ELIGIBLE, REFERENCE_NUMBER,
+                    claim.getReferenceNumber());
+                break;
+            case WAITING_TRANSFER:
+                appInsights.trackEvent(AppInsightsEvent.READY_FOR_TRANSFER, REFERENCE_NUMBER,
+                        claim.getReferenceNumber());
+                break;
+            default:
+                //Empty
+                break;
+        }
+    }
+
+    private void raiseAppInsightEventForOnlineOrOfflineDQ(Claim claim) {
```

Comment:
Since it's for both, can't this just be "...ForDQ"?
```suggestion
    private void raiseAppInsightEventForDQ(Claim claim) {
```